packages/cli/helpers/build.ts

```diff
@@ -1,7 +1,7 @@
 import type { BuildOptions } from '../../../helpers/compile/build'
 import { run } from '../../../helpers/compile/build'
 import { build } from '../../../helpers/compile/build'
-import { copySync } from 'fs-extra'
+import { copy } from 'fs-extra'
```

Comment:
Is there a deeper reason for that change or just style?

Reply:
`esbuild` plugins are `async`, so it's best not to block the main thread.